src/commands/test/lib/run-tests-command.ts

```diff
@@ -217,7 +217,9 @@ export abstract class RunTestsCommand extends AppCommand {
   }
 
   protected async cleanupArtifactsDir(artifactsDir: string): Promise<void> {
-    await pfs.rmDir(artifactsDir, true);
+    await pfs.rmDir(artifactsDir, true).catch(function(err){
+      console.warn(`Error ${err} while cleaning up artifacts directory ${artifactsDir}. Continuing without cleanup.`);
```

Comment:
Would it make sense to try to lead the user here? If files aren't being cleaned up - we could get them to check virus scan and local permissions issues (or anything else that we've discovered causes this in Intercom)?

Reply:
Absolutely - could you help me with some text?

Reply:
How about:
```
console.warn(`Error ${err} while cleaning up artifacts directory ${artifactsDir}. This is often due to files being locked or in use. Please check your virus scan settings and any local security policies you might have in place for this directory. Continuing without cleanup.`);
```

Reply:
👍